### ISSUE TITLE
readAnnotationElementValue is not implemented

### ISSUE THREAD

**Issue by didxga** [rmuller/infomas-asl#40]
First of all, thank you for this nice lib 👍 

It seems that `readAnnotationElementValue` is not implemented fully. It doesn't actually read the annotaion element value and store it. I think It's good feature to support as mostly of time when we scan annotator we will need element value along the way.

**Comment by rmuller** [comment]
You are right. This would be a nice addition. Are you using the `master` or `release-3.1` branch?

**Comment by didxga** [comment]
I checked both 3.0.X and 3.1.X, found that 3.1.X is still in snapshot.
